starter-code/src/App.js

```diff
@@ -1,18 +1,26 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import Beers from './components/Beers'
+import Random_Beer from './components/Random_Beer'
+import New_Beer from './components/New_Beer'
+import { Switch, Route, Link } from 'react-router-dom'
+
+
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+
+        {/* <Switch> */}
+        <Link to={`/beers`} >Beers</Link>
+        <Route path="/beers" component={Beers} />
+        <Link to={`/random-beer`} >Random_Beer</Link>
+        <Route path="/random-beer" component={Random_Beer} />
+        <Link to={`/new-beer`} >New_Beer</Link>
+        <Route path="/new-beer" component={New_Beer} />
+        {/* </Switch> */}
```

Comment:
you need to send params with your url, something like this <Route exact path='/:id' component={SingleBeer} />